f8a_worker/dispatcher/flows/osioAnalysisFlow.yml

```diff
@@ -4,5 +4,7 @@
       queue: '{DEPLOYMENT_PREFIX}_api_osioAnalysisFlow_v0'
       edges:
         - from:
-          to:
-            - 'dependency_tree'
\ No newline at end of file
+          to: 'GithubDependencyTreeTask'
+        - from: 'GithubDependencyTreeTask'
+          to: 'unknown_deps_fetcher'
```

Comment:
It's better to have a uniformity in names here

Reply:
@miteshvp Similar convention was followed in earstwhile stackAnalysisV2Flow as well, but I will surely make this uniform in an uncoming PR after the complete flow is implemented.